src/Styleguide/Components/__stories__/Tabs.story.tsx

```diff
@@ -58,12 +58,37 @@ storiesOf("Styleguide/Components", module).add("Tabs (Simple)", () => {
         </Tabs>
       </Section>
       <Section title="With separator">
-        <Tabs justifyContent="center" separator={<Box pr={"20px"}>*</Box>}>
+        <Tabs justifyContent="center" separator={<Box mx={1}>♞</Box>}>
           <Tab name="About" />
           <Tab name="Pricing" />
           <Tab name="Condition" />
         </Tabs>
       </Section>
+      <Section title="With superscript">
+        <Tabs justifyContent="center">
+          <Tab
+            name={
+              <React.Fragment>
+                Open<TabSuperscript>100</TabSuperscript>
+              </React.Fragment>
+            }
+          />
+          <Tab
+            name={
+              <React.Fragment>
+                Ready to ship<TabSuperscript>4</TabSuperscript>
+              </React.Fragment>
+            }
+          />
+          <Tab
+            name={
+              <React.Fragment>
+                Complete<TabSuperscript />
```

Comment:
Can `<Tabsuperscript>` be removed here?

Reply:
I left it intentionally as an example of having a tab without visible superscript and having the alignments perfect. I added a comment there.